### ISSUE TITLE
queries 

### ISSUE THREAD

**Issue by kamalibarani** [PingjunChen/KneeAnalysis#2]
hi sir, can i use  this code for mri scan images instead of xray images if yes how can i do.[https://github.com/PingjunChen/GradingKneeOA](url)

**Comment by CVAPPS24** [comment]
Hi! X-ray images are 2D, but MRI scans are 3D. Their input dimensions are different. If you use MRI slices, I think the code will work well. However, if your data is a 3D MRI scan, modification is required. 

How do you think, Pingjun? @PingjunChen

**Comment by PingjunChen** [comment]
Hi kamalibarani, I agree with Linlin @gll89. If you would like directly apply to 3D MRI, you may adapt the loss function in the 3D classification network with our proposed ordinal loss.

**Comment by kamalibarani** [comment]
Hi sir, if you have any documentation describing the procedure of project
please can you share me.

**Comment by PingjunChen** [comment]
Thanks for your interests, @kamalibarani  I believe our paper ["Fully Automatic Knee Osteoarthritis Severity Grading Using Deep Neural Networks with a Novel Ordinal Loss"](https://www.sciencedirect.com/science/article/abs/pii/S0895611118304956) has a relatively precise explanation of the whole procedure.  Besides, data and codes are shared here to facilitate reproduction.

**Comment by kamalibarani** [comment]
Tq sir. Iam not getting how to give paths for dataset. Can u help with that

**Comment by kamalibarani** [comment]
Hi sir, without cuda can't I implement this project
